### ISSUE TITLE
PHP 5.6: Numbers shown with scientific notation even if explicitly formatted as text

### ISSUE THREAD

**Issue by bhargav960143** [PHPOffice/PhpSpreadsheet#681]
This is:

```
- [X ] a bug report
```

### What is the expected behavior?
Export value as text

test_5.6.36.xlsx

```
89898989898958584854
```

### What is the current behavior?
Exported value as number in scientific notion

test_5.6.36.xlsx

```
8.9869E+18
```
""

### What are the steps to reproduce?

```php
<?php

require __DIR__ . '/vendor/autoload.php';

$data = array('number' => '89898989898958584854');
$spreadsheet = new \PhpOffice\PhpSpreadsheet\Spreadsheet();
$sheet = $spreadsheet->getSheet(0);
$sheet->fromArray($data, null, 'A1');
$sheet->getStyle('A1')
    ->getNumberFormat()
    ->setFormatCode(\PhpOffice\PhpSpreadsheet\Style\NumberFormat::FORMAT_TEXT);

$writer = new \PhpOffice\PhpSpreadsheet\Writer\Xlsx($spreadsheet);
$filename = 'test_' . PHP_VERSION . '.xlsx';
$writer->save($filename);
exit;
```

### Which versions of PhpSpreadsheet and PHP are affected?

phpspreadsheet: ^1.4
PHP 5.6.36
Tested on Windows English and Excel 2007 English. (Working)
Tested on Linux 2017 x86_64 (Host: x86_64-redhat-linux-gnu) (Not Working)

- Linux system is my server hosting system (Already configured system requirements.)

### Required Format

![required_format](https://user-images.githubusercontent.com/4817599/45732499-3fb75e00-bbfa-11e8-904f-a781a3493d51.png)

### Current Format

![current_format](https://user-images.githubusercontent.com/4817599/45732500-3fb75e00-bbfa-11e8-9ada-5383309ae0d3.png)

**Comment by kalabro** [comment]
As a workaround, I set format to `NumberFormat::FORMAT_NUMBER` and it is displayed properly (as a number still).

**Comment by oleibman** [comment]
Should be resolved by PR #4527. Removing stale label, leaving issue closed.
